installer.py

```diff
@@ -10,7 +10,7 @@
         "--add-data",
         "captures;captures",
         "--add-data",
-        "tft_bot/config.yaml;tft_bot",
+        "tft_bot/resources;tft_bot/resources",
```

Comment:
Does this entry need to be here twice? 🤔

Reply:
What do you mean by twice? It's `source_dir;target_dir`. It should be the same so we can reference it by the same path in code regardless of if we run from .py or from .exe

Reply:
What do you mean by twice? It's source_dir;target_dir. It should be the same so we can reference it by the same path in code regardless of if we run from .py or from .exe

Reply:
Ahh perf, it's been a long while since I looked at this code so that makes sense 👌